bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/CiphertextDecryptionTally.cs

```diff
@@ -46,15 +80,18 @@ public CiphertextDecryptionTally(CiphertextTally tally)
 
         foreach (var guardian in guardians)
         {
-            _guardians.Add(guardian.OwnerId, guardian);
+            _guardians.Add(guardian.GuardianId, guardian);
         }
 
         foreach (var ballot in ballots)
         {
             AddBallot(ballot);
         }
+        DecryptionState = DecryptionState.PendingGuardianShares;
     }
 
+    #region Misc Admin
+
     public void AddBallot(CiphertextBallot ballot)
     {
```

Comment:
So we can either add all of the ballots at once, which will not be great for large numbers of ballots, or we can add them one at a time.  Can we add an AddBallots method that can take a list of ballots instead of just a single 1.  This way it can be a little better balanced.